src/Settings/components/baniLength.jsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { useState } from "react";
+import { Text } from "react-native";
+import { ListItem, BottomSheet, Avatar, Icon } from "@rneui/themed";
+import STRINGS from "../../common/localization";
+import styles from "../styles";
+import { setBaniLength, BANI_LENGTHS } from "../../common/actions";
+import { useSelector } from "react-redux";
```

Comment:
It is the best practice to import from 3rd party first and then internal ones, so {useSelector} import should go to line no. 5. This problem is also there in other files,

Reply:
Fixed